README.md

```diff
@@ -387,7 +408,28 @@ We provide multiple datasets, both as starting points for each of the competitio
 
 - Our final processed dataset, DCLM-Baseline, is available on Huggingface in both [zstd compressed jsonl](https://huggingface.co/datasets/mlfoundations/dclm-baseline-1.0) and [parquet](https://huggingface.co/datasets/mlfoundations/dclm-baseline-1.0-parquet) formats. The former version is also available on the CommonCrawl S3 bucket, accessed via the instructions [here](https://data.commoncrawl.org/contrib/datacomp/DCLM-baseline/index.html).
 
-- We also provide a version of our dataset that performs all the steps of our preprocessing except the final one (namely, the fasttext filtering). This version, called DCLM-RefinedWeb, is also available on the CommonCrawl S3 bucket, with instructions available [here](https://data.commoncrawl.org/contrib/datacomp/DCLM-refinedweb/index.html)
+- We also provide a version of our dataset that performs all the steps of our preprocessing except the final one (namely, the fasttext filtering). This version, called DCLM-RefinedWeb, is also available on the CommonCrawl S3 bucket, with instructions available [here](https://data.commoncrawl.org/contrib/datacomp/DCLM-refinedweb/index.html). For official competition subsets of DCLM-RefinedWeb, please download the following shards (written in brace notation).
```

Comment:
We also need to add for 400m-1x

Reply:
there unfortunately isn't an appropriate one for 400m-1x yet because the decision to use subsets of local shards was based on this being the smallest unit that deduplication was run on.  1B-1x already corresponds to 1 local shard so going smaller would require subsetting a local shard (which is trickier to make sure hasn't been deduplicated against outside documents)

Reply:
can add a note about this for now

Reply:
I think it's fine if one shard is used (same as in 1b-1x case)?